docs/developers/tutorial.md

```diff
@@ -627,7 +626,7 @@ Another way to run custom rules from source is to use the `github:org/repo`
 scheme.
 
 ```
-scalafix --rules=github:olafurpg/named-literal-arguments
+scalafix --rules=github:scalacenter/scalafix-named-literal-arguments
```

Comment:
isn't the removal of `scalafix` breaking that https://scalacenter.github.io/scalafix/docs/developers/tutorial.html#using-github ?

Reply:
you're right. It's not working at all. 
it needs the directory scalafix + it's not the name of the repo, but the name of the rule: https://raw.githubusercontent.com/scalacenter/named-literal-arguments/master/rules/src/main/scala/fix/NamedLiteralArguments.scala

Do I rollback and put everything inside the directory scalafix ? It's annoying to have this directory sometimes, since ideas don't recognize it's scala code.